.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -16,7 +16,7 @@ labels: "a-Docs"
 
 
 **What would you like added or modified in the documentation?**
-<!-- 
+<!--
```

Comment:
why this is involved in the PR?

Reply:
They were generating warnings in travis. So just fixed them up in this. But ya makes no sense to do in this PR. Will remove.